packages/aa/test/utils.js

```diff
@@ -2,6 +2,53 @@ const path = require('node:path')
 const { symlink } = require('node:fs/promises')
 const { existsSync } = require('node:fs')
 
+const translateHeapStats = (stats = []) => {
```

Comment:
```suggestion
/**
 * @param {{space_name: string, space_used_size: number}[]} stats
 */
const translateHeapStats = (stats = []) => {
```

or make a `@typedef` for this somewhere